main.go

```diff
@@ -85,6 +86,27 @@ func checkUnused(deps []depEntry, vd string) {
 	}
 }
 
+func checkLicense(deps []depEntry, vd string) {
+	for _, d := range deps {
+		dir := filepath.Join(vd, d.importPath)
+		files, err := ioutil.ReadDir(dir)
+		if err != nil {
+			// err can be nil for unused package
+			continue
+		}
+		licenseFiles := 0
+		for _, file := range files {
+			p := filepath.Join(dir, file.Name())
+			if isLicenseFile(p) {
+				licenseFiles++
+			}
+		}
+		if licenseFiles == 0 && verbose {
+			log.Printf("WARNING(verbose): package %s may lack license information", d.importPath)
```

Comment:
I guess this will depend on the other branch so that it exits with a non-zero code?